tests/data/BNU1/sub-0025864/ses-1/.gitattributes

```diff
@@ -0,0 +1,6 @@
+anat/sub-0025864_ses-1_T1w.nii.gz filter=lfs diff=lfs merge=lfs -text
```

Comment:
These test files were deleted in the current staging (they are currently in neuroparc!)

Reply:
Manually deleted /ndmg/tests/data